crates/sncast/src/starknet_commands/call.rs

```diff
@@ -18,9 +18,9 @@ pub struct Call {
     #[clap(long)]
     pub function: String,
 
-    /// Arguments of the called function (list of hex)
+    /// Arguments of the called function (a list of Cairo-like expressions or values serialized as felts)
```

Comment:
I don't think `Cairo-like expression` is a term all users would understand. Perhaps this should show an example, be explained or just reworded?

Reply:
Sure, I will rewrite it in other words